accounts/serializers.py

```diff
@@ -1,16 +1,36 @@
-import pytz
 from rest_framework import serializers
 
-from .models import Log
+from .models import Log, User
 
 
 class LogSerializer(serializers.HyperlinkedModelSerializer):
-    def validate_timezone(self, timezone_str):
-        if timezone_str not in pytz.all_timezones:
-            raise serializers.ValidationError(
-                f'Invalid timezone string {timezone_str}')
-        return timezone_str
-
     class Meta:
         model = Log
         fields = '__all__'
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
if this is only going to be user for registering users, rename it something like `UserCreateSerializer `